features/support/env.rb

```diff
@@ -43,13 +45,31 @@
   config.ignore_hidden_elements = false
 end
 
+SitePrism.configure do |config|
+  config.use_implicit_waits = false
+end
+
 Capybara.register_driver :selenium do |app|
-  profile = Selenium::WebDriver::Firefox::Profile.new
-  profile['browser.cache.disk.enable'] = false
-  profile['browser.cache.memory.enable'] = false
-  Capybara::Selenium::Driver.new(app, browser: :firefox, profile: profile)
+  Capybara::Selenium::Driver.new(app, browser: browser, profile: profile)
 end
 
-SitePrism.configure do |config|
-  config.use_implicit_waits = false
+private
+
+def profile
+  if chrome?
+    Selenium::WebDriver::Chrome::Profile.new
+  else
+    Selenium::WebDriver::Firefox::Profile.new.tap do |profile|
+      profile['browser.cache.disk.enable'] = false
+      profile['browser.cache.memory.enable'] = false
+    end
+  end
+end
+
+def browser
+  ENV['browser'].to_sym
```

Comment:
Maybe you could just set a default here if the environment variable is unset?  That would make it easier for most users.

Reply:
Agreed. I think in one of the commits I didn't copy over he specified it to default to firefox. Will add that in (Along with a rebase)

Check back say Monday?